app/views/api.js

```diff
@@ -6,18 +6,33 @@
 
 import asyncHtml from "choo-async/html";
 import dedent from "dedent";
-import fs from "graceful-fs";
-import raw from "choo/html/raw";
+import redirectOr404 from "../modules/redirectOr404";
+import headerBlockchain from "../components/api/header-blockchain";
+import headerSdk from "../components/api/header-sdk";
 
 const fetch = require("make-fetch-happen").defaults({ cacheManager: "./cache" });
 
-//  V A R I A B L E
-
-const apiScripts = "<script>" + fs.readFileSync("./app/components/client/api-scripts.js", "utf-8") + "</script>";
+/*
```

Comment:
Why is this here?

Reply:
this was left over from a conflicted rebased, removed